doc/examples/transform/plot_register_rotation.py

```diff
@@ -96,3 +96,117 @@
+# center. However, it is more often the case that there is also a translation
+# component to the difference between two images to be registered. One
+# approach to register rotation, scaling and translation is to first correct
+# for rotation and scaling, then solve for translation. It is possible to
+# resolve rotation and scaling differences for translated images by working on
+# the magnitude spectra of the Fourier-transformed images.
+
+from skimage.color import rgb2gray
+from skimage.filters import window, difference_of_gaussians
+from scipy.fftpack import fft2, fftshift
+
+angle = 24
+scale = 1.4
+shiftr = 30
+shiftc = 15
+
+image = rgb2gray(data.retina())
+translated = image[shiftr:, shiftc:]
+rotated = rotate(translated, angle)
+rescaled = rescale(rotated, scale)
+sizer, sizec = image.shape
+rts_image = rescaled[:sizer, :sizec]
+
+# When center is not shared, log-polar transform is not helpful!
+radius = 705
+warped_image = warp_polar(image, radius=radius, scaling="log")
+warped_rts = warp_polar(rts_image, radius=radius, scaling="log")
+tparams = register_translation(warped_image, warped_rts, upsample_factor=20)
+shifts = tparams[0]
+shiftr, shiftc = shifts[:2]
+klog = radius / np.log(radius)
+shift_scale = 1 / (np.exp(shiftc / klog))
+
+fig, axes = plt.subplots(2, 2, figsize=(8, 8))
+ax = axes.ravel()
+ax[0].set_title("Original Image")
+ax[0].imshow(image, cmap='gray')
+ax[1].set_title("Modified Image")
+ax[1].imshow(rts_image, cmap='gray')
+ax[2].set_title("Log-Polar-Transformed Original")
+ax[2].imshow(warped_image)
+ax[3].set_title("Log-Polar-Transformed Modified")
+ax[3].imshow(warped_rts)
+fig.suptitle('log-polar-based registration fails when no shared center')
+plt.show()
+
+print(f"Expected value for cc rotation in degrees: {angle}")
+print(f"Recovered value for cc rotation: {shiftr}")
+print()
+print(f"Expected value for scaling difference: {scale}")
+print(f"Recovered value for scaling difference: {shift_scale}")
+
+# Now try working in frequency domain
+# First, band-pass filter both images
+image = difference_of_gaussians(image, 5, 20)
+rts_image = difference_of_gaussians(rts_image, 5, 20)
+
+# window images
+wimage = image * window('hann', image.shape)
+rts_wimage = rts_image * window('hann', image.shape)
+
+# work with shifted FFT magnitudes
+image_fs = np.abs(fftshift(fft2(wimage)))
+rts_fs = np.abs(fftshift(fft2(rts_wimage)))
+
+# Create log-polar transformed FFT mag images and register
+shape = image_fs.shape
+radius = shape[0] // 8  # only take lower frequencies
+warped_image_fs = warp_polar(image_fs, radius=radius, output_shape=shape,
+                             scaling='log', order=0)
+warped_rts_fs = warp_polar(rts_fs, radius=radius, output_shape=shape,
+                           scaling='log', order=0)
+
+warped_image_fs = warped_image_fs[:shape[0] // 2, :]  # only use half of FFT
+warped_rts_fs = warped_rts_fs[:shape[0] // 2, :]
+tparams = register_translation(warped_image_fs, warped_rts_fs,
```

Comment:
Could you please use here `registration.phase_cross_correlation` since the name changed?